src/pages/dan/Main/Feed/Comment/CommentList.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import Comment from './Comment';
+import './CommentList.scss';
+
+class CommentList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentList: [],
+      comment: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/commentData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          commentList: data,
+        });
+      });
+  }
+
+  textChange = event => {
+    this.setState(
+      {
+        comment: event.target.value,
+      },
+      () => {
+        this.commentValid();
+      }
+    );
+  };
+
+  commentValid() {
+    const { comment } = this.state;
+    return comment.length > 0 ? true : false;
+  }
+
+  addComment = () => {
+    let token = localStorage.getItem('우리토큰') || '';
+    fetch('http://10.58.7.150:8000/posts/comment', {
+      headers: {
+        Authorization: token,
+      },
+      method: 'POST',
+      body: JSON.stringify({
+        comment_post: this.state.comment,
+        post_id: '1',
+      }),
```

Comment:
- 보통 `method -> headers -> body ` 순서로 작성합니다.
```suggestion
      method: 'POST',
      headers: {
        Authorization: token,
      },
      body: JSON.stringify({
        comment_post: this.state.comment,
        post_id: '1',
      }),
```